curling/test.py

```diff
@@ -128,8 +131,8 @@ def setUp(self):
         self.api = MockAPI('')
 
     def test_get_one(self):
-        eq_(self.api.services.settings('APPEND_SLASH').get_object(),
-            samples['GET:/services/settings/APPEND_SLASH/'])
+        eq_(self.api.services.setting.get_object(),
+            {'key': 'ABSOLUTE_URL_OVERRIDES'})
```

Comment:
This `test_get_one` was not ran before, because the name clashes with another `test_get_one` test below. I tried changing the name and it didn't pass at all:
`AssertionError: {u'key': u'APPEND_SLASH'} != {'content': '{"key": "APPEND_SLASH"}'}`

Ideas?